sonar-plugin-api/src/main/java/org/sonar/api/batch/Decorator.java

```diff
@@ -24,9 +24,12 @@
 
 /**
  * @since 1.10
+ * @deprecated since 5.2 there's no more decorator on batch side
+ * TODO add link to no API
```

Comment:
We should add a link to the new (not no !) API, but this API is not yet available